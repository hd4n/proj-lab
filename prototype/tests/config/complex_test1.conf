#item_nev id
#equipment, material
@items
am a1
am a2
am a3
nu n1
nu n2
nu n3

#effekt_nev id duration
@effects
fo d1 3

#agent_nev id effekt_id
@agents
vi vi1 d1

#kod_nev id agent_id aminocost nucleocost
@codes
co cd1 vi1 3 3

#mezo_nev id [tartalom]
@fields
em e1
em e2
wa w1
la l1

#mezo_id1 mezo_id2
#ketiranyu kapcsolat
@neighbors
e1 l1
l1 w1
w1 e2

#id akt_mezo [item_id1;item_id2...] [material_id1;material_id2...] [effect_id1;effect_id2...]
@virologists
v1 e1 null null null
v2 e2 null null null