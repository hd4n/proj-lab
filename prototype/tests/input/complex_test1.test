load complex-test1.conf
getstats v1
getstats v2
step v1 l1
nextround
interact v1 readcode
step v1 w1
nextround
interact v1 collect_material
craft v1 cd1
useagent v1 v2 vi1
nextround
getstats v1
getstats v2