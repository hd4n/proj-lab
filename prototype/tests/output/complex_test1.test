Load: sikeres

stats: v1
	fi_e1
	di_null
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_null
	ma_null
	ag_null
	eq_null
	eqa_null
	ef_null

stats: v2
	fi_e2
	di_null
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_null
	ma_null
	ag_null
	eq_null
	eqa_null
	ef_null
step: v1 l1
nextround: sikeres
interact: v1 l1
step: v1 w1
nextround: sikeres
interact: v1 w1
craft: v1 cd1
useagent: v1 v2 vi1
nextround: sikeres

stats: v1
	fi_w1
	di_w1
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_0
	neededNucleo_1
	co_cd1
	ma_null
	ag_null
	eq_null
	eqa_null
	ef_null

stats: v2
	fi_e2
	di_l1
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_null
	ma_null
	ag_null
	eq_null
	eqa_null
	ef_d1
