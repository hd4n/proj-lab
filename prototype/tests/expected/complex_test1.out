Load: sikeres
stats v1:
fi_e1
eq_null
ma_null
ef_null
co_null

stats v2:
fi_e2
eq_null
ma_null
ef_null
co_null

step: v1 l1

nextround

interact: v1 readcode

step: v1 w1

nextround

interact: v1 collect_material

craft: c1 cd1 vi1

useagent:
v1+v2
dance

nextround

stats v1:
fi_w1
eq_null
ma_null
ef_null
co_null

stats v2:
fi_e2
eq_null
ma_null
ef_d1
co_null

stats map:
e1 - l1
l1 - w1
w1 - e2